lib/colorado_lottery.rb

```diff
@@ -0,0 +1,19 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants, :winners, :contestants
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    return true if contestant.age < 18 && contestant.game_interests.include?(game.name)
```

Comment:
The logic is flipped here and should be `return true if contestant >= 18 && contestant.game_interests.include?(game.name)`